src/transformers/models/chinese_clip/image_processing_chinese_clip.py

```diff
@@ -246,25 +247,23 @@ def preprocess(
 
         images = make_list_of_images(images)
 
-        if not valid_images(images):
+        if images and not valid_images(images):
```

Comment:
Same here - I'll stop commenting on each case but comment applies to the other similar parts of the diff :)

Reply:
Yes, I'll make a batch replace :) thanks a lot!